### PR TITLE
Add a template for new issues.

### PR DESCRIPTION
**Pull request by thomasvl**
Per: https://help.github.com/articles/creating-an-issue-template-for-your-repository/

### CODE REVIEW

File: .github/issue_template.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Please be sure to include:
+
+* what version of Xcode you are using: `xcodebuild -version`
```
Comment (allevato) [inline .github/issue_template.md]: We should generalize this for Linux as well.

Maybe ask "What operating system you are using" and then "What version of Xcode (if macOS)"?

Comment (thomasvl) [reply .github/issue_template.md]: Ok, i'm confused, I amended the commit, but the PR won't pick it up…

Comment (thomasvl) [reply .github/issue_template.md]: And github finally realizes it…

File: .github/issue_template.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Please be sure to include:
+
+* what operating system you are developing on (linux or macOS)
+
+* for macOS, what version of Xcode you are using (`xcodebuild -version`)
+
+* what branch/tag of SwiftProtobuf you are using (1.0.0, etc.)
+
+* what version of Swift is your code set to compile with
```
Comment (tbkka) [inline .github/issue_template.md]: Maybe just "the version of Swift you are using" with a callout to `swift --version`

In any case, I would put this before the SwiftProtobuf version.  There's a natural order here:  either OS -> dev tools -> project   or   project -> dev tools -> OS.

File: .github/issue_template.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+Please be sure to include:
+
+* what operating system you are developing on (linux or macOS)
```
Comment (tbkka) [inline .github/issue_template.md]: "your operating system (Linux or macOS, including the version)"

File: .github/issue_template.md (hunk 2 above)
Comment (tbkka) [inline .github/issue_template.md]: I'm not sure if it's worth specifically asking people to try to:
 * copy/paste the specific compiler error or warning they're concerned about
 * Include a short piece of code that demonstrates a problem they're seeing

**Comment by tbkka** [comment]
It's certainly better than what we have.  Go for it!

File: .github/issue_template.md (hunk 3 above)
Comment (thomasvl) [reply .github/issue_template.md]: done.

File: .github/issue_template.md (hunk 2 above)
Comment (thomasvl) [reply .github/issue_template.md]: The catch is the version of swift they use does not have to the same version that the library is compiled with (or that `swift --version`) will report.  Xcode settings, Package.swift info, etc all seems to come into play.

About the only thing I thought I was sure about was `swift --version` would only tell us if they had a custom version installed, but even if they do, that doesn't mean Xcode ends up using it.

Anyway, generally tweaked.

Comment (thomasvl) [reply .github/issue_template.md]: Added some more.

**Pull request merged**